src/ReadingBuddy.java

```diff
@@ -735,10 +734,10 @@ public static void searchForBookBuddyMenu(){
                         2. Enter a book (title and author)
                         3. Select a book from the list of books
                         4. View all \"Book Buddy Wanted\" Posts
+                        5. Manage your \"Book Buddy Wanted\" posts
```

Comment:
added the manage your own posts option under the "Find a Book Buddy" main menu option. It fits thematically, but I don't think it would be an intuitive place to look for this function if I hadn't coded it... It can be moved! Lmk if either of you have thoughts on where would be a good/intuitive place for it to go (wording on some other menu item can also be added or changed to make it more clear (or we just add a comment in the final code acknowledging it and that it would be easy to solve if we had actual UI?))